APIs/schemas/constraints_supported.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": [
+    "parameter_constraints"
+  ],
+  "properties": {
+    "parameter_constraints": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^urn:x-nmos:cap:"
+      },
+      "contains": {
+        "type": "string",
+        "enum": [
+          "urn:x-nmos:cap:meta:preference",
+          "urn:x-nmos:cap:meta:enabled",
+          "urn:x-nmos:cap:format:media_type",
+          "urn:x-nmos:cap:format:grain_rate",
+          "urn:x-nmos:cap:format:frame_width",
+          "urn:x-nmos:cap:format:frame_height",
+          "urn:x-nmos:cap:format:interlace_mode",
+          "urn:x-nmos:cap:format:colorspace",
+          "urn:x-nmos:cap:format:color_sampling",
+          "urn:x-nmos:cap:format:component_depth",
+          "urn:x-nmos:cap:format:channel_count",
+          "urn:x-nmos:cap:format:sample_rate",
+          "urn:x-nmos:cap:format:sample_depth"
+        ]
```

Comment:
Missing urn:x-nmos:cap:format:transfer_characteristic

I think that those are also required for audio urn:x-nmos:cap:transport:packet_time, urn:x-nmos:cap:transport:max_packet_time

Also this one for compatibility of flow urn:x-nmos:cap:transport:st2110_21_sender_type

Reply:
I would require as little mandatory Parameter Constraints supported as possible but it may make the API completely useless. So I consider a single-codec straight-forward baseband-to-network converter as a target device with minimal functionality. If it only can packetize a baseband stream, determine its parameters and modify own Base EDID, it makes sense to me to keep only Parameter Constraints which can directly effect EDID to let the physical source try another format.

The list turns out even shorter than the original one:
`urn:x-nmos:cap:meta:label`,
`urn:x-nmos:cap:meta:preference`,
`urn:x-nmos:cap:meta:enabled`,
`urn:x-nmos:cap:format:grain_rate`,
`urn:x-nmos:cap:format:frame_width`,
`urn:x-nmos:cap:format:frame_height`,
`urn:x-nmos:cap:format:interlace_mode`,
`urn:x-nmos:cap:format:color_sampling`,
`urn:x-nmos:cap:format:component_depth`,
`urn:x-nmos:cap:format:channel_count`,
`urn:x-nmos:cap:format:sample_rate`,
`urn:x-nmos:cap:format:sample_depth`

Otherwise, such a device can't use IS-11 unless its vendor implements converting the baseband stream before sending to the network.

Reply:
We agreed about this list on the activity group call on 2020/11/08. I made a commit with it.